site/docs/contribute/logging.md

```diff
@@ -36,15 +36,15 @@ Log levels should be used as follows:
 ## JSON Logging Standard Format
 All Fybrik components should generate logging information in a standard format. This information will be used by different actors for different purposes, so as much relevant information as possible needs to be captured in a consistent format.
 
-We list all mandatory and optional fields to be used by all Fybrik components. In addition to the fields we list, Fybrik components may include extra fields as needed.
+We list all mandatory and optional fields to be used by all Fybrik components. In addition to the fields we list, Fybrik components may include extra fields as needed. The fields in this section are typically generated by the logging libraries.
```

Comment:
Please move the comment about the mandatory fields to the mandatory field section.  The optional fields are not automatically generated by log libraries